usaspending_api/search/v2/views/search.py

```diff
@@ -644,6 +381,8 @@ def post(self, request):
         for m in models:
             if m['name'] in ('award_type_codes', 'fields'):
                 m['optional'] = False
+            if m['name'] == 'award_type_codes':
+                m['enum_values'] = m['enum_values'] + ['no intersection']  # handles an edge case from the Frontend
```

Comment:
Let's move 'no intersection' back to the award_type_codes model in usaspending_api/core/validator/award_filter.py